ga4gh/cli.py

```diff
@@ -816,3 +818,37 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+def run_test(testObj, test):
+    """
+    :param testObj: The object under test
+    :param test: The test method to run
+    :return: Either None for success, or a tuple of the test name and raised
+    exception
+    """
+    try:
+        getattr(testObj, test)()
+    except AssertionError as e:
+        return test, e
+    else:
+        return None
+
+
+def configtest_main(parser=None):
+    testConfig = configtest.TestConfig(parser)
```

Comment:
Maybe we could use `unittest.loadTestsFromModule` or similar to discover the tests, so we don't have to register each one?